app/schemas/models/district.schema.js

```diff
@@ -0,0 +1,30 @@
+const schema = require('./../schemas')
+
+const DistrictSchema = schema.object({}, {
+  name: schema.shortString(),
```

Comment:
I am making the assumption that on the server we can leave off the `plugins.NamedPlugin` from the model and it won't get the pre-save hook that slugifies the name. Thus this property can be duplicated without issue.